nav2_simple_commander/package.xml

```diff
@@ -12,6 +12,7 @@
   <exec_depend>nav2_msgs</exec_depend>
   <exec_depend>action_msgs</exec_depend>
   <exec_depend>lifecycle_msgs</exec_depend>
+  <exec_depend>aws_robomaker_small_warehouse_world</exec_depend>
```

Comment:
Remove-  I know its actually a dependency, but the package is flaky at being released and I don't want this to block Nav2 binary releases, even if folks need to build themselves locally